public/testrun.html

```diff
@@ -65,11 +63,24 @@ <h1>
   <div id='testrun-loading-div'></div>
 </div>
 <div id='main-div'></div>
+
+<ul>
+  <li><span class='code-snippet'>npm run dev</span> (server process and client rebundling) must be active to (re)compile</li>
+  <li><span class='code-snippet'>/specs</span> response data is needed to render the cards.</li>
+  <li>Clicking on a card title or entry opens a URL with glob patterns to filter the spec filename that would run.</li>
+  <li>If tests are missing and console shows error with /specs?exclude= request, check your <code>sjpp/serverconfig.json</code> to make sure <span class='code-snippet'>"debugmode":true</span> is set at root level.</li>
+  <li>To run tests on commandline, do <span class='code-snippet'>npm run test:integration --workspace=client</span> at /sjpp/proteinpaint/ directory.
+    Or, from the client dir, <span class='code-snippet'>npm run test:integration</span> or <span class='code-snippet'>npm run test:unit</span>
+  </li>
+<ul>
+
 <script>
-const time = Date.now()
+
+const time = Date.now() // why need to add time to href?
```

Comment:
comment on why time is needed?

Reply:
I put the `time` URL param to force the browser to always reload a test glob pattern, I think early in the switch to esbuild, the test would not refresh to use a new client bundle. I tested just now without the `time` URL param, and edited client code and spec files, and I cannot replicate the previous issue, so I removed it for now as it doesn't seem to be needed anymore.